server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -134,6 +135,9 @@ def fulfill(uttr: Utterance) -> PopulateState:
     # only do toolformer rig updates in single sv case
     if state.uttr.mode == QueryMode.TOOLFORMER_RIG:
       _update_chart_vars_for_rig(state)
+    # only do toolformer rig updates in single sv case
```

Comment:
nit: duplicate comment?  also drop "rig" since

Reply:
removed and updated the comment above to say toolformer rig and rag